fabtests/common/shared.c

```diff
@@ -3059,6 +3064,25 @@ void ft_parse_hmem_opts(int op, char *optarg, struct ft_opts *opts)
 	}
 }
 
+void parse_opts_range (char* optarg)
+{
+       int start, inc, end;
+       int i, range_test_cnt;
+       sscanf (optarg, "r:%d,%d,%d", &start, &inc, &end);
+
+       /* populate the array */
+       range_test_cnt = (end - start) / inc + 1;
```

Comment:
This calculation is off if end doesn't align with start + some multiple of inc.  E.g. start = 1, end = 2, and inc = 2 has 1 valid value.  There needs to be some adjustment to end so that it falls on a valid value.  Btw, I like the idea of this supporting decreasing sizes, but that's added complexity that may not be worth adding right now.  If not, then we'll want a sanity check that end >= start